lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -117,6 +117,45 @@ $notes-annotator-background-dark: rgba(122,122,122,0.6); // taken from annotator
   .annotator-listing {
     padding: 0 !important;
     margin: 0 !important;
+
+    .annotator-widget {
+
+      // using annotatorJS triangle styling for adder
+      &:after {
```

Comment:
Almost all of these properties are repeated in two places. Can you create a notes-bubble Sass extend in this file and extend it here?